eng/build.yml

```diff
@@ -1,53 +1,54 @@
 parameters:
+  # Agent OS identifier and used as job name
   agentOs: ''
-  queue: {}
+
+  # Agent pool
+  pool: {}
+
+  # Additional variables
   variables: {}
 
-phases:
-- template: /eng/common/templates/phases/base.yml
+  # Build strategy - matrix
+  strategy: {}
```

Comment:
change this to `''` so that it aligns properly with the condition below that is checking to see if "strategy" is an empty string or not.  I'm not sure if the comparison of `{}` resolves to `''` or not.

Reply:
Done